### ISSUE TITLE
Reserving Prefixes

### ISSUE THREAD

**Issue by TylerLeonhardt** [PowerShell/PowerShellGallery#73]
We should give larger customers of the Gallery the ability to reserve a module prefix so that users can be sure that anything with that prefix is that developer.

Here are some examples:

Azure PowerShell - `Az.*`
AWS PowerShell - `Az.Tools.*`

**Comment by bpayette** [comment]
Yes please! We (AWS PowerShell) have already had squatters in the `AWS.Tools.*` prefix.
